Source/Models/SwishError.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Argo
+
+public enum SwishError {
+  case SessionError(NSError)
+  case ServerError(code: Int, json: AnyObject)
+  case JSONSerializationError(NSError)
+  case ArgoError(DecodeError)
```

Comment:
Do you think it's a smell that every case here ends in `Error`? It feels like duplication to me.